### ISSUE TITLE
Prettify Script

### ISSUE THREAD

**Issue by speroamare** [jneale/Xplore#57]
Can we get a UI action/Keyboard short cut to prettify the highlighted script. Hopefully match the OOB Script field shortcut Shift+Tab.

**Comment by mgrak** [comment]
Ctrl-Shift-Tab is the hot key used in the script field type to prettify, oob. My vote is matching what the rest of the platform already uses for the short cut.    On Monday, August 29, 2022 at 07:53:49 AM PDT, James Neale ***@***.***> wrote:  
 
 


Closed #57 as completed.

**Comment by jneale** [comment]
Ctrl-Shift-Tab changes browser tabs for me. Ctrl-A then Shift-Tab is what works for formatting.

**Comment by mgrak** [comment]
You are right sorry :) my muscle memory is bettor then my recall. Ctrl-A and then Shift-Tab is right
    On Monday, August 29, 2022 at 12:07:21 PM PDT, James Neale ***@***.***> wrote:  
 
 


Ctrl-Shift-Tab changes browser tabs for me. Ctrl-A then Shift-Tab is what works for formatting.
